SwiftDDP/DDPExtensions.swift

```diff
@@ -257,9 +257,13 @@ extension DDPClient {
         method("login", params: NSArray(arrayLiteral: params)) { result, error in
             guard let e = error where (e.isValid == true) else {
                 
-                if let user = params["user"],
-                    let email = user["email"] {
+                if let user = params["user"] {
+                    if let email = user["email"] {
                         self.userData.setObject(email, forKey: DDP_EMAIL)
+                    }
+                    if let username = user["username"] {
```

Comment:
Setting the username here is very important otherwise logout will crash since it doesn't have the email field and `self.user()` will return nil.